frontend/src/App.tsx

```diff
@@ -413,6 +420,10 @@ async function GameController(
   socket.on('playerFinished', (finishedPlayer: ServerPlayer, score: number, gaveUp: boolean) => {
     const player = Player.fromServerPlayer(finishedPlayer);
     displayPlayerFinishedToast(player, score, gaveUp);
+    dispatchAppUpdate({
+      action: 'updatePlayerHasCompletedMaze',
+      player,
+    });
```

Comment:
correct me if im wrong but this only fires for the other player in a game? so any outsiders wont know if a player has finished the maze or not

Reply:
![image](https://user-images.githubusercontent.com/43456581/114741543-dbf56300-9d18-11eb-80fa-083a32446923.png)

Reply:
Oh I thought we agreed that it would emit to everybody, seems like that's not what the backend currently does -- I'll update it